runners/core-java/src/main/java/org/apache/beam/runners/core/SplittableParDo.java

```diff
@@ -400,6 +399,15 @@ public void setOutputWindowedValue(OutputWindowedValue<OutputT> outputWindowedVa
       this.outputWindowedValue = outputWindowedValue;
     }
 
+    public DoFn<InputT, OutputT> getFn() {
+      return fn;
+    }
+
+    @Setup
+    public void setup() throws Exception {
+      invoker = DoFnInvokers.invokerFor(fn);
```

Comment:
Done. It lead to a slightly bigger refactoring and a rephrasing of the commit message, so I also added tests.